src/main/jbake/content/models.ad

```diff
@@ -70,3 +213,11 @@ That can easily be done with:
 
 More information about release signing and verifying signatures can
 be found https://www.apache.org/dev/release-signing.html[here].
+
+# SourceForge Models
+
+The models on Sourceforge for 1.5.0 are http://opennlp.sourceforge.net/models-1.5/[found here]
```

Comment:
Maybe use https link (nit picking - IDE linter is now detecting http links as low issues, not important)

Reply:
Looks like https://opennlp.sourceforge.net/models-1.5/ doesn't work. Do you know if there's a different https address for sourceforge?

Reply:
Oh, you are right. I think they don't offer an HTTPS version at all. Sorry!